zendeskTicketNumber: 7545
zendeskLink: https://sourcegraph.zendesk.com/agent/tickets/7545
title: small group of repos failing to fetch/clone
Application engineer: Jason Harris
Customer: Netflix 
Date: Apr 8
Version: 3.37.0â€‹
Deployment: pure-docker
External Services: BITBUCKETSERVER
Auth Providers: http-header
Slack Links: https://sourcegraph.slack.com/archives/C012V43N8E7/p1649437212426909
GitHub Issue Link: https://github.com/sourcegraph/customer/issues/824
Doc Update Link: N/A
Summary: For a while now, a small number of repos in Netflix instance have been failing to fetch/clone. The customer asked if we could do some more digging on why this is the case. Netflix's logs show the following errors:
    clone failed. Output: \nfatal: remote error: upload-pack: not our ref 0000000000000000000000000000000000000000\nfatal: git upload-pack: not our ref 0000000000000000000000000000000000000000: exit status 128
The not our ref error is consistent in this issue.
-   Tried netflix's internal /refresh/ API. Made no difference.
-   Sent the following docs to manually refresh repos: https://docs.sourcegraph.com/admin/how-to/remove-repo. Still failing to fetch/clone.
-   I suspected this was related to their unique deployment/bitbucket refs, which is out of my expertise, so I filed an issue with repo management.
-   Disabled ref caching on Bitbucket server, then tried the steps in the link above again. No dice.
The customer said this is enough to confirm to him that the problem seems to be related to the data for those particular repos.
"Given the above, I feel confident there is not a lot Sourcegraph can do for those repos, so I'll investigate more on my side). Given that I think you can deprioritize this investigation..."
https://docs.sourcegraph.com/admin/how-to/remove-repo
    clone failed. Output: \nfatal: remote error: upload-pack: not our ref 0000000000000000000000000000000000000000\nfatal: git upload-pack: not our ref 0000000000000000000000000000000000000000: exit status 128
